app/components/holdings/search_location_component.rb

```diff
@@ -8,13 +8,30 @@ def initialize(holding_hash)
 
     private
 
+      attr_reader :holding_hash
+
       def location
+        return library_in_use if library_in_use
         helpers.holding_library_label(holding_hash)
       end
 
+      def library_in_use
+        location_code = holding_hash['location_code']
+        library_in_use_locations = {
+          'arch$pw': "Archictecture (Remote Storage)",
+          'eastasian$pl': "East Asian (Remote Storage)",
+          'engineer$pt': "Engineering (Remote Storage)",
+          'firestone$pb': "Firestone (Remote Storage)",
+          'firestone$pf': "Firestone (Remote Storage)",
+          'lewis$pn': "Lewis (Remote Storage)",
+          'lewis$ps': "Lewis (Remote Storage)",
+          'mendel$pk': "Mendel (Remote Storage)",
+          'stokes$pm': "Stokes (Remote Storage)"
+        }.freeze
+        library_in_use_locations[location_code.to_sym] if library_in_use_locations.key?(location_code.to_sym)
```

Comment:
Could this be simplified to not use the `if`?  I think it would have the same behavior (i.e. returning `nil` if the hash does not contain the key)

```suggestion
        library_in_use_locations[location_code.to_sym]
```